packages/x-charts/src/internals/useAnimate.ts

```diff
@@ -0,0 +1,91 @@
+'use client';
+import * as React from 'react';
+import { ANIMATION_DURATION_MS, ANIMATION_TIMING_FUNCTION_JS } from './animation';
+import { Transition } from './Transition';
+import { shallowEqual } from './shallowEqual';
+import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect';
+
+export function useAnimate<Props extends {}, Elem extends Element>(
```

Comment:
Can we move `animation.ts`, `Transition.tsx`, `useAnimate.ts` and `useAnimate.test.tsx` to a joint folder? `internals/useAnimate/` probably?

Reply:
Sure 👍